### PR TITLE
[Intl] [ResourceBundle] refactor LanguageBundle

### PR DESCRIPTION
**Pull request by beebop1032**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?     | no
| New feature? | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by beebop1032** [comment]
First contribution enjoy!

**Comment by ro0NL** [comment]
Can you update the base branch of your PR to master? The diff includes 2.7-master currently ;-)

File: src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php [hunk 1]
```diff
@@ -81,7 +81,7 @@ public function getLanguageNames($displayLocale = null)
         try {
             return $this->getNames($displayLocale);
         } catch (MissingResourceException $e) {
-            return array();
+            return [];
```
Comment (ro0NL) [inline src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php]: SF dont use this syntax (yet); so for consistency it's better to this all at once, or on a per-component base.

File: CHANGELOG-3.0.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
  * bug #19127 [Form] Add exception to FormRenderer about non-unique block names (enumag)
  * bug #19118 [Process] Fix pipes cleaning on Windows (nicolas-grekas)
  * bug #19128 Avoid phpunit 5.4 warnings on getMock (2.7+) (iltar)
- * bug #19114 [HttpKernel] Dont close the reponse stream in debug (nicolas-grekas)
+ * bug #19114 [HttpKernel] Dont close the response stream in debug (nicolas-grekas)
```
Comment (ro0NL) [inline CHANGELOG-3.0.md]: These file are auto-generated on release, the typo comes from the PR actually ;-) see https://github.com/symfony/symfony/pull/19114

File: src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php (hunk 1 above)
Comment (beebop1032) [inline src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php]: Ok thanks, so i close this PR

Comment (ro0NL) [inline src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php]: Guess so :+1: thx for taking time though :)

**Pull request closed without merging**